tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,168 @@
+## **Create New Identity With Attributes Using Local Fabric CA**
+`15-20 mins`
+
+## Learning Objectives
+
+* Create our smart contract
+* Create a new identity with attributes using Local Fabric CA.
+* Use the newly created identity and implement attribute-based access control.
+    * Package the smart contract
+    * Install the smart contract
+    * Instantiate the smart contract
+
+> Note: This tutorial assumes you have created the `Jack` identity by following the <a href='./createNewIdentity.md'>previous tutorial </a>.
+
+---
+<details>
+<summary><b>1. Creating our smart contract</b></summary>
+
+For the purposes of this tutorial, we'll use TypeScript as the example language.
+
+> In VS Code, every command can be executed from the Command Palette (press `Ctrl+Shift+P`, or `Cmd+Shift+P` on MacOS). All of this extension's commands start with `IBM Blockchain Platform:`. In the tutorial steps, we'll explain where to click in the UI, but look out for comment-boxes like this one if you want to know the Command Palette alternatives.
```

Comment:
why is this note just in this one, surely this should be something to add to the very first tutorial and then you wouldn't need this here

Reply:
Fixed - added the above note at the start of the first tutorial and removed from this tutorial.